tests/unit/test_redfish.py

```diff
@@ -481,14 +479,23 @@ def mock_get_response(uri):
         mock_redfish_obj.get.assert_any_call("/redfish/v1/Systems/s2/Processors")
         mock_redfish_obj.get.assert_any_call("/redfish/v1/Systems/s2/Processors/p21")
 
+    @patch("prometheus_hardware_exporter.collectors.redfish.redfish_client")
+    def test__storage_root_uri(self, mock_redfish_client):
```

Comment:
We should test different pattern with different expected_uri. (standard and non-standard.)